chart/templates/gitops.yaml

```diff
@@ -1278,10 +1315,10 @@ spec:
           value: {{ .Values.gotk.env.noProxy }}
         {{- end }}
         {{- if .Values.gotk.image.registry }}
-        image: {{ .Values.gotk.image.registry }}/fluxcd/helm-controller:v0.1.1
+        image: {{ .Values.gotk.image.registry }}/fluxcd/helm-controller:v0.1.3
         {{- else }}
-        image: ghcr.io/fluxcd/helm-controller:v0.1.1
-        {{- end }}
+        image: ghcr.io/fluxcd/helm-controller:v0.1.3
+        {{- end }}        
```

Comment:
Is this extra tab intentional?  If not, you might want to just check on what it does to your YAML formatting...

Reply:
it works but will remove that, thanks